src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/service/AlunoService.java

```diff
@@ -1,36 +1,83 @@
 package com.projetoGerenciamentoCurso.gerenciamentoCurso.service;
 
+import javax.transaction.Transactional;
+
 import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.projetoGerenciamentoCurso.gerenciamentoCurso.*;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.GerenciamentoCursoApplication;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.AlunoDto;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.models.Aluno;
+import com.projetoGerenciamentoCurso.gerenciamentoCurso.repository.AlunoRepository;
 
 import lombok.AllArgsConstructor;
 
 @Service
 @AllArgsConstructor
+@Transactional
 public class AlunoService {
 
 	private final RabbitTemplate rabbitTemplate;
-
-	public void retornaAluno(AlunoDto aluno) {
+	
+	@Autowired
+	private AlunoRepository alunoRepository;
+	
+	public Aluno retornaAluno(AlunoDto aluno) {
 
 		rabbitTemplate.convertAndSend(GerenciamentoCursoApplication.EXCHANGE_NAME,
 				GerenciamentoCursoApplication.ROUTING_KEY, aluno);
+		
+		Aluno alunoModel = alunoRepository.save(criarAlunoModel(aluno));
+		
+		return alunoModel;
 
 	}
 
-	public void deletarAluno(AlunoDto aluno) {
+	public Aluno deletarAluno(AlunoDto aluno) {

```

Comment:
Por ser a deleção de uma entidade, o método não deve ter retorno
```suggestion
	public void deletarAluno(AlunoDto aluno) {
```

Reply:
feito